apps/stats-dapp/src/containers/AuthoritiesTable/AuthoritiesTable.tsx

```diff
@@ -200,6 +201,10 @@ export const AuthoritiesTable: FC<AuthoritiesTableProps> = ({
     getFilteredRowModel: getFilteredRowModel(),
   });
 
+  const tableIsLoading = useMemo(() => {
```

Comment:
Why don't we utilize the `isLoading` boolean which is returned by the `useAuthorities` hook.